llpc/context/llpcCompiler.cpp

```diff
@@ -846,7 +846,10 @@ Result Compiler::buildPipelineWithRelocatableElf(Context *context, ArrayRef<cons
       auto data = reinterpret_cast<const char *>(elfBin.pCode);
       elf[stage].assign(data, data + elfBin.codeSize);
       LLPC_OUTS("Cache hit for shader stage " << getShaderStageName(static_cast<ShaderStage>(stage)) << "\n");
-      stageCacheAccesses[stage] = CacheAccessInfo::CacheHit;
+      if (userShaderCache == nullptr)
+        stageCacheAccesses[stage] = CacheAccessInfo::InternalCacheHit;
```

Comment:
nit: Can you turn it into a conditional assignment? I mean `shaderCacheaccess[stage] = userShaderCache ? A : B`.

Reply:
done